pkg/informer/informer.go

```diff
@@ -129,10 +167,9 @@ func (d *DynamicDiscoverySharedInformerFactory) Listers() (listers map[schema.Gr
 		return
 	}
 
-	for gvr := range d.gvrs {
+	for gvr, informer := range d.informers {
```

Comment:
if we are adding cluster scoped resources now, should we maybe have a `NamespacedListers()`, `ClusterScopedListers()` ?